cumulus/polkadot-omni-node/lib/src/command.rs

```diff
@@ -148,6 +148,15 @@ pub fn run<CliConfig: crate::cli::CliConfig>(cmd_config: RunConfig) -> Result<()
 				node.prepare_revert_cmd(config, cmd)
 			})
 		},
+		Some(Subcommand::ChainSpecBuilder(cmd)) => {
+			if let Err(err) = cmd.run() {
+				eprintln!("Error executing chain-spec-builder: {}", err);
+				return Err(sc_cli::Error::Application(err.into()));
+			}
+
+			Ok(())
```

Comment:
Something like this won't look better?

```suggestion
			cmd.run()
			     .map_err(|err| { 
			         eprintln!("Error executing chain-spec-builder: {}", err)
			         Error::Application(err.into())
			     )
```

Reply:
addressed in https://github.com/paritytech/polkadot-sdk/pull/7619/commits/b4c89c42b03b30816091b752395ce22a1728c7ce